.circleci/config.yml

```
@@ -513,6 +513,52 @@ jobs:
             bin/ci-test-route "ashley" "ok" "/__heartbeat__"
 
 
+  # Test the bootstrap playbook on the "nextcloud" application.
+  # We also need redis and mailcatcher for this application
+  # nota bene: we use a real OpenShift cluster installed in CircleCI's VM.
+  test-bootstrap-nextcloud:
+    machine:
+      docker_layer_caching: false
+
+    working_directory: ~/fun
+
+    steps:
+      - checkout
+      - *attach_workspace
+      - *docker_load
+      - *ci_env
+      - *install_openshift_cluster
+      - *configure_openshift_cluster
+      - *run_openshift_cluster
+
+      - run: 
+          name: Rebuild arnold image once project created to use the modified user_id variable
+          command: |
+            bin/ci-ansible-playbook create_project.yml "redis,mailcatcher,nextcloud"
+            export USER_ID=$(oc get project ci-eugene -o json | jq '.metadata.annotations."openshift.io/sa.scc.uid-range"' | cut -d'/' -f 1 | cut -d'"' -f 2)
```

Maybe use `sed` instead?

```suggestion
            export USER_ID=$( \
              oc get project ci-eugene -o json | \
              jq '.metadata.annotations."openshift.io/sa.scc.uid-range"' | \
              sed -r "s|([0-9]+)/([0-9]+)|\\1|" \
            )
```

---

I made a little modification, using `-r` option with `jq` to remove double quotes.